R/tm_data_table.R

```diff
@@ -1,34 +1,34 @@
-#' Data Table Viewer `teal` Module
+#' Data table viewer module
 #'
-#' A data table viewer shows the data using a paginated table.
-#' specifically designed for use with `data.frames`.
-#' @md
+#' Module provides a dynamic and interactive way to view `data.frame`s in a `teal` application.
+#' It uses the `DT` package to display data tables in a paginated, searchable, and sortable format,
```

Comment:
Maybe a link do `DT` package?

Reply:
I have linked the functions from `DT` that are mentioned in the parameter description. In description we did not link package directly (functions or topic only)

Reply:
[25b47e4](https://github.com/insightsengineering/teal.modules.general/pull/667/commits/25b47e40d945bcfc97e4be841ae69ffdb710b985)